### DISCUSSION TITLE
I am trying to load fullpageOS to display a dashboard from monda.com. but i am not able to get the aspect ratio right. Even after changing the config file its not working. What can be done.? I am using a raspberry pi 4

### DISCUSSION THREAD

**Discussion by AmalAnup** [guysoft/FullPageOS#593] [Q&A]
#### What were you doing?

[Please be as specific as possible here]

#### Version of FullPageOS?

[Can be found in /etc/fullpageos_version ALWAYS INCLUDE.]

#### Screenshot(s) showing the problem:

[If applicable. Always include if unsure or reporting UI issues.]

**Comment by guysoft** [comment]
Converting to a discussion, it not a bug or feature.
